bookkeeper-server/src/main/java/org/apache/bookkeeper/client/DistributionSchedule.java

```diff
@@ -197,6 +204,11 @@ public WriteSet copy() {
      */
     AckSet getAckSet();
 
+    /**
+     * Returns an ackset object useful to wait for all bookies in the ensemble,
+     * responses should be checked against this.
+     */
+    AckSet getAckSetForForceLedger();
```

Comment:
`getEnsembleAckSet(long entryId)`. same reason as `getEnsembleWriteSet`.

Reply:
seems to me that entryId is not neeeded in this case, I will rename to getEnsembleActSet